Projects/Features/Scene/GroupDetailScene/Vote/Vote/VoteView.swift

```diff
@@ -36,27 +35,42 @@ public struct VoteView: View {
           .padding(.trailing, 16)
           .padding(.top, 27)
         }
-        
-        Spacer()
-      }
 
-      VStack(spacing: 0) {
+        TitleView(name: store.userInfo.nickname)
         
-        TitleView(name: store.name)
+        Spacer()
         
         VoteSwipeView(
-          imageURLs: store.imageURLs,
-          imageCount: store.imageCount,
+          voteOptions: store.voteOptions,
+          voteOptionCount: store.imageCount,
           swipeDirection: store.swipeDirection,
           swipeAction: { index, swipeDirection in
             store.send(.cardSwiped(index, swipeDirection))
           }
         )
-        .padding(.bottom, 68)
+        
+        Spacer()
         
         VoteButtonView(store: store)
+        
+        Spacer()
+      }
+      
+      if store.isNeedGuideView && !store.guideViews.isEmpty {
+        VoteGuideView(
+          guideViews: store.guideViews,
+          swipeAction: {
+            store.send(.guideViewSwiped)
+          }
+        )
+        .animation(.easeIn)
```

Comment:
P2:
`'animation' was deprecated in iOS 15.0: Use withAnimation or animation(_:value:) instead.`